src/views/components/CurrentLineup.vue

```diff
@@ -5,19 +5,19 @@
 				<span> Pre Team Spacer </span>
 			</div>
 			<div class="col-1 animal-col">
-				<pet-display pet-type="Beaver" />
+				<PetCard pet-type="Beaver" />
```

Comment:
use lower case kebab case for tags, there is one weird interaction it prevents that i can't remember off the top of my head

Reply:
Will fix in later release (target date 2025)